admin/view-translations-media.php

```diff
@@ -4,30 +4,33 @@
  * Needs WP 3.5+
  *
  * @package Polylang
+ *
+ * @var PLL_Admin_Classic_Editor $this
+ * @var PLL_Language             $lang
+ * @var int                      $post_ID
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Don't access directly
-}
+defined( 'ABSPATH' ) || exit; // Don't access directly
```

Comment:
```suggestion
defined( 'ABSPATH' ) || exit; // Don't access directly.
```

Do we add this comment every time or not? 
If so, add it where it's missing.

Reply:
This is a protection against direct execution.
But files without side effects do not need it. A file with only `class Name { ... }` is parsed but execution only involves defining a class, no instantiation.

Reply:
As @screenfeed said to me: we also put it when a class extends another class, implements a interface or use a trait. If there is a fatal error it could have a full path disclosure which display the webroot path.

For consistency we decided to put it everywhere not to forgot it and if a class definition change in the future.

In this PR, I put it only in file I needed to modify. The goal isn't to modify all the polylang files.

Reply:
> Do we add this comment every time or not? If so, add it where it's missing.

Concerning the comment. I modified this line manually and the comment was here before the modification and then I kept it like this. The comment is useless. We can remove it everywhere.

Done for this file in 7eb6ea6e27ab6daf91e444750528577a1b7afa4e